src/util/get-github-client-config.ts

```diff
@@ -66,9 +69,9 @@ export async function getGitHubHostname(jiraHost: string, gitHubAppId: number) {
  * get all installation or get more info for the app
  */
 export async function createAppClient(logger: Logger, jiraHost: string, gitHubAppId: number | undefined): Promise<GitHubAppClient> {
-	const gitHubClientConfig = await getGitHubClientConfigFromAppId(gitHubAppId);
+	const gitHubClientConfig = await getGitHubClientConfigFromAppId(gitHubAppId, jiraHost);
 	return await booleanFlag(BooleanFlags.GHE_SERVER, false, jiraHost)
-		? new GitHubAppClient(logger, gitHubClientConfig.baseUrl)
+		? new GitHubAppClient(logger, gitHubClientConfig.baseUrl, gitHubClientConfig.appId.toString(), gitHubClientConfig.privateKey)
```

Comment:
wouldn't it make more sense that all `GithubAppClient` take in a `GitHubClientConfig` parameter?